build-logic/plugins/src/main/kotlin/Extention.kt

```diff
@@ -0,0 +1,34 @@
+import java.io.ByteArrayOutputStream
+
+/*
+ * Wire
+ * Copyright (C) 2024 Wire Swiss GmbH
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ */
+
+fun String.execute(): Process {
```

Comment:
I think this one is already defined under, we might reuse ? Since we have a project object where this is used?

https://github.com/wireapp/wire-android/blob/e1478880ce53d8154540aa5d17c85d4fc311ea92/buildSrc/src/main/kotlin/ProjectExtensions.kt#L52-L62